### PR TITLE
Documentation extendedCallWindow de_DE

### PR DESCRIPTION
**Pull request by Notme112**
I have made some Documentation for the extetded call window. All the other i do if i find a little bit time.

### CODE REVIEW

File: src/modules/extendedCallWindow/docs/de_DE.md [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
-## benötigte Fahrzeuge verschönern
+## Benötigte Fahrzeuge verschönern
+Wer kennt es nicht, grade wenn viele Fahrzeuge gefordert sind, ist die Nachforderung ziemlich unübersichtlicht und wenn man einmal etwas übersieht, dauert es noch länger, bis der Einsatz abgearbeitet wird. Dieses Feature hilft dir dabei, bei jedem Einsatz den Überblick zu bewahren, indem es die noch benötigeten fahrzeuge übersichtlich als Tabelle darstellt.
+![Tabelle mit zusaätzlich benötigten Fahrzeugen](fahrzeugeVerschoenern.png)
+
+### Tooltips
+Mit dieser Einstellung werden zu den Buttons von "Benötigste Fahrzeuge verschönern" kleine Hinweisfelder ergänzt, die erscheinen, sobal man die Maus kurz auf dem jeweiligen Button ruhen lässt.
+
+### Maximalbesetzung
+Manchmal wird am Einsatzort ja auch eine bestimmte Personalmenge gebraucht. Dieses Feature ergänzt die Personalrechnung, indem es davon ausgeht, dass jedes fahrzeug mit der Maximalen Personalmenge ausrückt.
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/docs/de_DE.md:22]: switch those two settings as they are also switched in settings.

File: src/modules/extendedCallWindow/docs/de_DE.md [hunk 2]
```diff
@@ -1,30 +1,62 @@
...
+![Zurücksetzen-Button](fahrzeugwahlReset.png)
 
-## unpassende AAOs ausgrauen
+## Unpassende AAOs ausgrauen
+Suchst du immer ewigkeiten die passende AAO, weil du für jeden Einsatz eine andere AAO hast? Dieses Modul wirst du lieben. Es garut alle AAO's, in denen nicht mindestens ein Teil des Einsatzstichwortes vorkommt, aus.
+![ausgegraute AAO's](aao5.png)
 
 ## AAO-Anfahrtszeit anzeigen
+Diese Einstellung zeigt an, wie lange das am längsten brauchende Fahrzeug der AAO braucht. Erscheint, die Maus kurz auf der AAO ruhen lässt.
+![AAO mit angezeigter Anfahrtszeit des am längsten brauchenden Fahrzeuges](aao6.png)
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/docs/de_DE.md:59]: Is this the LSSM Version or ingame Version?
Doesn't LSSM show the time in a green box?

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/modules/extendedCallWindow/docs/de_DE.md (hunk 2 above)
Comment (Notme112) [reply src/modules/extendedCallWindow/docs/de_DE.md:59]: This ist the Version of the LSSM, its in the AAO direcly

Comment (jxn-30) [reply src/modules/extendedCallWindow/docs/de_DE.md:59]: Oh yeah, I see.
Does not look that beatiful xD

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**